src/tactic/command/login_group_trigger.py

```diff
@@ -46,14 +47,36 @@ def execute(my):
 
         login_group = login_group_name.lower()
         login_group = login_group.replace(" ", "_")
+        login_group_code = sobj.get_value('login_group')
 
         sobj.set_value('login_group', login_group)
         sobj.set_value('code', login_group)
 
         sobj.commit(triggers=False)
 
+        my.update_related(login_group, login_group_code)
 
+    def update_related(my, login_group, prev_login_group):
 
+        login_in_groups = Search.eval("@SOBJECT(sthpw/login_in_group['login_group','%s'])" % prev_login_group)
 
```

Comment:
use the regular to avoid potential syntax error if they use ' in the group name
search = Search('sthpw/login_in_group')
search.add_filter('login_group', prev_login_group)
login_in_groups = search.get_sobjects()